st2common/bin/st2ctl

```diff
@@ -87,6 +87,11 @@ function st2start() {
   for COM in ${COMPONENTS}; do
     service_manager ${COM} start
   done
+  PYTHON_VERSION=$(/opt/stackstorm/st2/bin/python --version 2>&1 | awk '{print $2'} | awk -F"." '{print $1}')
+  if [ "${PYTHON_VERSION}" = "2" ]; then
+    echo -e "\e[33mDeprecation warning: Support for python 2 will be removed in future StackStorm releases. Please ensure that all packs used are python 3 compatible. Your StackStorm installation may be upgraded from python 2 to python 3 in future platform releases. It is recommended to plan the manual migration to a python 3 native platform, e.g. Ubuntu 18.04 LTS or CentOS/RHEL 8. \e[0m\n"
```

Comment:
That looks nice! 

Can we include the Deprecation warning for the st2ctl reload/register actions as well to make it even more visible people can't ignore?

Reply:
That should be easy enough to add - will do.

Reply:
Nothing major, but the second awk is redundant. A single call to `awk` can give the python major version, as follows

```
$ python --version 2>&1 | awk -F"[. ]" '{print $2}'
3
```

Reply:
Cool - will fix that up as well. And also try and remember the bit of ST2 I stole it from!

Reply:
@armab Added warning into st2ctl reload and st2ctl restart-component. (start and restart covered by logic already there). 
@nzlosh Thanks for the awk comment, much neater now, that's much neater (I'd stolen it from st2-self-check - but I won't patch that up under this PR).